src/adaptive_card.rs

```diff
@@ -949,12 +951,19 @@ pub enum ContainerStyle {
 #[allow(missing_docs)]
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub enum Spacing {
```

Comment:
All of these can be done with 
`#[serde(rename_all = "camelCase")]`

Reply:
They can't - it needs to be an alias, because apparently the webex API uses both `small` and `Small` - unless there's an alias version of `rename_all`?